eGovPDX/portlandor
# POWR-2185 Update morsel to match news teaser/featured needs

**This PR accomplishes the following:**

- Update the design system morsel pattern to match the latest comps in Figma
- Update the design system demos and documentation to match the new design
- IMPLEMENT: the new morsel pattern within News Teaser & Featured displays (below)
- Exported configuration to the News Teaser & Featured displays (date format updates)

**To Test:**

Visit the below URLs to verify the correct implementation of the new morsel pattern:

- https://powr-2185-portlandor.pantheonsite.io/omf (must be logged in)
- https://powr-2185-portlandor.pantheonsite.io/news
- https://powr-2185-portlandor.pantheonsite.io/novel-coronavirus-covid-19
